tests/console/console.go

```diff
@@ -111,11 +111,11 @@ func RunCommand(vmi *v1.VirtualMachineInstance, command string, timeout time.Dur
 		&expect.BSnd{S: "echo $?\n"},
 		&expect.BCas{C: []expect.Caser{
 			&expect.Case{
-				R: shellSuccess,
+				R: regexp.MustCompile(ShellSuccess),
```

Comment:
It just hurts my eye to see this repeated on each call to this function. Can you maybe call it `regexShellSuccess` and keep it there?

Reply:
It is used once for Fail once for Success. I don't see any reason to exract somethong that is used only once to a const/variable.

Reply:
My point is that the compilation now occurs each time this command is called. It is enough to compile it once and not on every call.

Reply:
done.